components/toggle.tsx

```diff
@@ -2,33 +2,36 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 
-export default function Toggle({onToggle, toggled, onText, offText, disabled}) {
+type Props = {
+    onToggle: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+    toggled: boolean,
+    disabled?: boolean,
+    onText: React.ReactNode,
+    offText: React.ReactNode,
+}
+
+export default function Toggle(props: Props) {
+    const {onToggle, toggled, disabled, onText, offText} = props;
     return (
         <button
             type='button'
             onClick={onToggle}
             className={`btn btn-lg btn-toggle ${toggled && 'active'} ${disabled && 'disabled'}`}
             aria-pressed={toggled ? 'true' : 'false'}
-            autoComplete='off'
         >
             <div className='handle'/>
             {text(toggled, onText, offText)}
         </button>);
 }
 
-function text(toggled, onText, offText) {
+function text(
```

Comment:
You can maintain everything in the same line here, probably is more consistent with the rest of our code base.